### PR TITLE
specs-go: add label to config.go

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

follow up https://github.com/opencontainers/image-spec/pull/371

### CODE REVIEW

File: specs-go/v1/config.go [hunk 1]
```diff
@@ -45,6 +45,9 @@ type ImageConfig struct {
 
 	// WorkingDir sets the current working directory of the entrypoint process in the container.
 	WorkingDir string `json:"WorkingDir,omitempty"`
+
+	// Labels contains arbitrary metadata for the container.
+	Labels map[string]struct{} `json:"labels,omitempty"`
```
Comment (wking) [inline specs-go/v1/config.go]: The spec calls for [`map[string]string`][1].

And we'll need JSON Schema work too, although that doesn't have to happen in this PR.

[1]: https://github.com/opencontainers/image-spec/pull/371/files#diff-c9c91c29b41257aea3a3403cc606ad99R139

Comment (coolljt0725) [inline specs-go/v1/config.go]: @wking  yes, it's `map[string]string`. 　

> And we'll need JSON Schema work too, although that doesn't have to happen in this PR.

working on it

**Comment by vbatts** [comment]
@coolljt0725 rebase please?

**Comment by coolljt0725** [comment]
@vbatts  rebased

**Comment by vbatts** [comment]
LGTM

**Comment by philips** [comment]
LGTM

**Pull request merged**